src/dht.cpp

```diff
@@ -1245,6 +1243,9 @@ Dht::announce(const InfoHash& id, sa_family_t af, std::shared_ptr<Value> value,
             callback(false, {});
         return;
     }
+
+    storageStore(id, value, created);
+
```

Comment:
This change has been introduced by `bbd1200`. You should have cherry-picked it. Now, it won't be clear in the log if index has this fix or not.